packages/run-protocol/test/amm/constantProduct/test-checkInvariants.js

```diff
@@ -110,6 +119,7 @@ function checkGetOutput(t, args, result) {
 
 const testGetInputPrice = (t, inputs, runIn) => {
   const args = runIn ? prepareRUNInTest(inputs) : prepareRUNOutTest(inputs);
+  // @ts-ignore
```

Comment:
add a few words about why `@ts-ignore` is appropriate here?

Reply:
done